fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerTextView.kt

```diff
@@ -308,13 +312,14 @@ internal class PeoplePickerTextView : TokenCompleteTextView<IPersona> {
 
     override fun canDeleteSelection(beforeLength: Int): Boolean {
         // This method is called from keyboard events so any token removed would be coming from the user.
-        shouldAnnouncePersonaRemoval = true
```

Comment:
Same here?

Reply:
Since announcement of span removal and addition is on by default, so setting it to true is redundant.